.github/workflows/build.yaml

```diff
@@ -21,20 +21,32 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       release_run: ${{ steps.setvar.outputs.release_run }}
+      branch:   ${{ steps.which-branch.outputs.branch }}
+      relnotes: ${{ steps.which-branch.outputs.relnotes }}
     env:
       # Build with a tag like "Second_Life#abcdef0" to generate a release page
       # (used for builds we are planning to deploy).
+      # Even though inputs.release_run is specified with type boolean, which
+      # correctly presents a checkbox, its *value* is a GH workflow string
+      # 'true' or 'false'. If you simply test github.event.inputs.release_run,
+      # it always evaluates as true because it's a non-empty string either way.
       # When you want to use a string variable as a workflow YAML boolean, it's
       # important to ensure it's the empty string when false. If you omit || '',
-      # its value when false is "false", which is interpreted as true.
-      RELEASE_RUN: ${{ (github.event.inputs.release_run || github.ref_type == 'tag' && startsWith(github.ref_name, 'Second_Life')) && 'Y' || '' }}
+      # its value when false is "false", which (again) is interpreted as true.
+      RELEASE_RUN: ${{ (github.event.inputs.release_run != 'false' || (github.ref_type == 'tag' && startsWith(github.ref_name, 'Second_Life'))) && 'Y' || '' }}
```

Comment:
good catch, sorry about that

Reply:
I don't mean to throw shade! Even if you echo `github.event.inputs.relase_run` to stdout, you see `false` either way. There's no way to directly ask the workflow engine for the _type_ of that variable, and since you explicitly state that it's `boolean`, it's entirely reasonable to think that type might persist. I only got here because empirically it wasn't working, and empirically this change made it better.

Reply:
yeah no offense taken, just lamenting how hard it is to test this CI stuff before it lands